guide/install.md

```diff
@@ -69,12 +69,15 @@ docker run --name slidev --rm -it \
     --user node \
     -v ${PWD}:/slidev \
     -p 3030:3030 \
+    -e NPM_MIRROR="https://registry.npmmirror.com" \
     tangramor/slidev:latest
 ```
 
-If your work folder is empty, it will generate a template `slides.md` and other related files under your work folder, and launch the server on port `3030`.
+***Note***: You can use `NPM_MIRROR` to specify a npm mirror to speed up the installation process.
 
-You can access your slides from `http://localhost:3030/`
+If your work folder is empty, it will generate a template `slides.md` and other related files under your work folder, and launch the server on port `3030`. 
+
+You can access your slides from http://localhost:3030/
```

Comment:
The quotes are intentionally, otherwise VitePress will fail as it to be a deadlink